graph.go

```diff
@@ -195,16 +195,21 @@ func (c *GraphCmd) Execute(args []string) error {
 }
 
 func relPath(base, path string) string {
-	rp, err := filepath.Rel(base, path)
+	pathWithoutSymlinks, err := filepath.EvalSymlinks(path)
```

Comment:
In this case, though, let's say the user is just running `src` manually (with no Sourcegraph installation). If you run it in symlinked repository, will the relative paths be relative to all the source/underlying paths? That would be surprising to me as a user.